scripts/publish-for-forge.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
+
+set -eu
+
+clean() {
+  # TODO: use regex instead
+  find . -maxdepth 1 \
+    ! -path . \
+    ! -path ./.git \
+    ! -path "./.git/*" \
+    ! -path ./packages \
+    ! -path ./LICENSE \
+    -exec rm -rf {} +
+}
+
+main() {
+  git fetch origin main
+  latest_main_commit_msg=$(git log origin/main -1 --pretty=%B)
+
+  git checkout -b forge
+  git fetch origin forge
+  git pull origin forge --rebase || true
+
+  clean
+
+  # http://mywiki.wooledge.org/BashFAQ/001
+  # https://github.com/koalaman/shellcheck/wiki/SC2012
+  find packages -maxdepth 1 -mindepth 1 -printf '%P\n' | while read -r pkg; do
+    mkdir "$pkg"
+    mv "packages/$pkg/src" "$pkg"
+    mv "packages/$pkg/README.md" "$pkg"
+    git add "$pkg"
+  done
+
+  rm -fr "packages"
+
+  git commit -am "$latest_main_commit_msg"
```

Comment:
this script is executed in the `release` workflow...   
which is triggered on tags push events on main...  
Tags are created with `yarn version:bump`...  
Which always creates a `chore(pkg): v(version)` commit...  

So on the `forge` branch we will only have a clean succession  of `chore(pkg): v(version)` commit messages that mirrors our npm release workflow, making it easy for devs to checkout/select a specific contracts package version even if they install with `forge install`